src/Bundle/Controller/Workflow.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\Controller;
+
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Symfony\Component\Workflow\Registry;
+
+final class Workflow implements StateMachineInterface
+{
+    /** @var Registry */
+    private $registry;
+
+    public function __construct(Registry $registry)
+    {
+        $this->registry = $registry;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
What are these inheritdoc comments used for? I have always wondered, never found an answer

Reply:
It means the documentation is inherited from parent.

Reply:
Yes. My comment was too vague. I meant: What tooling uses these 'inheritdocs'? I never add these and I haven't heard anyone wanting to add them - on the contrary in fact.

Reply:
Just saw this commit right now: https://github.com/Sylius/SyliusResourceBundle/commit/369bfadc73c6b49011ab058397e4791418a6ab1c. Looks like I am not the only one wondering

Reply:
It's used by IDEs and code assistance tools like PHPSTORM and https://serenata.gitlab.io/ and by documentation generators like https://www.phpdoc.org/.
You can see it in [PSR-19 Draft](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#42-using-the-inheritdoc-inline-tag-to-augment-a-description), which accompanies [PSR-5 Draft](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md) in an attempt to make a standard for PHP doc blocks. 
Similar notation is used in Java, JS, XML and probably more.

Reply:
![image](https://user-images.githubusercontent.com/7114562/96220981-6ccdca80-0f92-11eb-9e4b-d8fd1e83dda1.png)
Here's an example where it can be useful, but adding it everywhere makes no sense.